app/views/home.scala.html

```diff
@@ -5,25 +5,45 @@
 @import db.ModelService
 @import db.impl.access.UserBase
 @import models.project.Project
-@import ore.{OreConfig, Platforms}
 @import ore.Platforms.Platform
 @import ore.project.Categories._
 @import ore.project.ProjectSortingStrategies.{values, _}
 @import ore.project.{Categories, ProjectSortingStrategies}
+@import ore.{OreConfig, Platforms}
+@import play.api.Configuration
 @import security.NonceFilter._
+@import scala.collection.JavaConversions._
 
+@import scala.collection.mutable
 @import scala.util.Random
+
 @(models: Seq[Project], visibleCategories: Option[Array[Category]], query: Option[String], page: Int,
         sort: ProjectSortingStrategy, platform: Option[Platform])(implicit messages: Messages, flash: Flash,
         request: Request[_], service: ModelService, config: OreConfig, users: UserBase)
 
 @projectRoutes = @{controllers.project.routes.Projects}
 
 @randomSponsor = @{
-    val logos = config.sponge.getStringList("sponsors.logos").get
-    val urls = config.sponge.getStringList("sponsors.urls").get
+    val rawLogos: List[Configuration] = config.sponge.getConfigList("sponsors").get.toList
+    val logos: mutable.MutableList[Configuration] = mutable.MutableList()
+
+    var currentIndex = 1
+    rawLogos.foreach(logo => {
+        val count: Int = rawLogos.size - rawLogos.indexOf(logo)
```

Comment:
This will make MC Pro Hosting appear 7 times more than YourKit, and I don't think that's a fair distribution. Instead, we should balance these out by donation amount/sponsorship tier.

Reply:
Actually, the sponsorship images are ordered appropriately for how much they've donated, MC Pro Hosting being the most and YourKit being the least I think. @Dockter will need to confirm this?

Reply:
They are in the correct _order_, but the chance of them appearing needs to be proportional to their contribution amount.

Reply:
Also also thought about this.. With we could introduce 4 tiers. Gold, Silver, Bronze and Normal or something similar.

Reply:
Maybe for now we should have them all be equal until we have a system in place to determine contribution distribution.